test/testdata/packager/export_for_test/foo/foo.rb

```diff
@@ -18,26 +18,25 @@ def self.stub_stuff!; end
   # via import Opus::Foo::Bar
   Opus::Foo::Bar::BarClass
   Test::Opus::Foo::Bar::BarClassTest
-# ^^^^ error: Unable to resolve constant `Test`
-
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ error: Used test-only constant `Test::Opus::Foo::Bar::BarClassTest` in non-test file
```

Comment:
We use the phrase "test-only" in two contexts:

- test-only import (`test_import`)
- constant defined inside the `Test::` namespace

What do you think about changing the non-`test_import` error message to mention something like

```
`Test::Opus::Foo::...` is defined in a test namespace and cannot be referenced within non-test `Opus::Foo` namespace
```

?

(Also, it's not that `foo/foo.rb` is a test or non-test file is it? It's that `Opus::Foo` is a non-test namespace? lmk if i'm mistaken).

Reply:
Ok so it looks like my understanding was wrong, and that we actually do report this using the file information, not the namespace.

Question for my own understanding: what is the relationship between a test file and the `Test::` namespace?

Reply:
If a file's extension is `.test.rb` or has `/test/`, it must be defined in the test namespace.

Reply:
I like your suggested phrasing for this error more, I'll switch it over.